src/util/manager/firestore.ts

```diff
@@ -2,16 +2,258 @@ import Bot from "../../api/bot";
 import { settings } from "../../settings";
 import { Firestore } from "@google-cloud/firestore";
 import Manager from "../../api/manager";
+import {
+  Circle,
+  CircleData,
+  Coper,
+  CoperData,
+  response,
+  ResponseType,
+  RRMessageData,
+  rrMessageData,
+  Task,
+} from "../../api/schema";
+import { z } from "zod";
+
+export const cacheTypes = z.object({
```

Comment:
trying to understand the difference btwn cache and schema in this context. schema is the firebase side name and cache is the local name? If thats the case, why not unify the two?